### PR TITLE
Lint models/update.go, release.go & webhook_slack.go

### PR DESCRIPTION
**Pull request by Bwko**

### CODE REVIEW

**Comment by strk** [comment]
LGTM

**Comment by andreynering** [comment]
LGTM

File: models/update.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ import (
 	"code.gitea.io/gitea/modules/log"
 )
 
+//UpdateTask defines an UpdateTask
```
Comment (tboerger) [inline models/update.go]: missing space behind `//`, on multiple places below as well.

**Comment by lunny** [comment]
resolved #70

**Pull request merged**